osu.Game/Skinning/SkinModelManager.cs

```diff
@@ -262,5 +262,7 @@ public void Save(Skin skin)
                 s.Hash = ComputeHash(s);
             });
         }
+
+        public override bool IsAvailableLocally(SkinInfo model) => false;
```

Comment:
bit of a weird default to leave without a TODO at least

Reply:
Think it would be better to default to `true`? I've changed to that and added a comment.

Reply:
`true` is better for sure as far as I'm concerned.